### PR TITLE
Feature/layout #19 

### PR DESCRIPTION
**Pull request by flxtagi**
userHeader and split screen

### CODE REVIEW

File: src/components/MainLayout.tsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+import { LayoutProps } from '../types/layoutProps';
+import { useSession } from 'next-auth/react';
+import { TUser } from '../types';
+
+const MainLayout: LayoutProps = ({ children }) => {
+	const { data: session } = useSession();
+	const currentUser: TUser | undefined = session?.user;
+
+	// TODO some fallback for failed login
+	const emptyUser = {
```
Comment (rudigier) [inline src/components/MainLayout.tsx:12]: Default data should not be part of layout.

File: src/components/LoginLayout.tsx [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+	return (
+		<>
+			<Head>
+				<link rel="icon" href="/favicon.ico" />
+				<title>Login to Mumble</title>
+			</Head>
+			<div className="SplitScreen grid grid-rows-1 grid-cols-2 md:grid-cols-1 w-screen min-h-screen">
+				<header className="column-start-1 column-span-1 row-start-1 row-span-1 flex items-center justify-center bg-gradient-to-tl from-violet-600 to-pink-500">
+					<div className="w-8/12 text-pink-300 text-center">
+						<svg
```
Comment (rudigier) [inline src/components/LoginLayout.tsx:21]: svg to a external file?

**Comment by rudigier** [comment]
@flxtagi The favicon to public/

File: src/components/LoginLayout.tsx (hunk 2 above)
Comment (flxtagi) [reply src/components/LoginLayout.tsx:21]: thats a great idea!

File: src/components/MainLayout.tsx (hunk 1 above)
Comment (flxtagi) [reply src/components/MainLayout.tsx:12]: sure we need some fallback somewhere for the not logged in.

Comment (flxtagi) [reply src/components/MainLayout.tsx:12]: removed

**Review by rudigier — APPROVED** [review]

**Review by rudigier — APPROVED** [review]
Looks great. Thanks!

**Pull request merged**